### DISCUSSION TITLE
Accept the new Terms of Service to upgrade your Google Analytics for Firebase property

### DISCUSSION THREAD

**Discussion by jpinto321** [invertase/react-native-firebase#6024] [Q&A] [answered]
Hi everyone,

On our Firebase dashboard, it shows a message saying we need to upgrade our Google Analytics for Firebase property.
As soon as we try to accept it, we get to go through a wizard that shows some instructions where we need to add some Firebase/Google analytics changes to our project.

We already have Firebase set up with react-native-firebase, so I'm wondering if I should just skip these changes or are they needed? For instance, I do not see implementation platform('com.google.firebase:firebase.bom.....') in our project, though it's working just fine.

Do we need to do anything here, or are these instructions just for new projects that have never set up Firebase yet?

![Screen Shot 2022-01-19 at 3 17 38 PM](https://user-images.githubusercontent.com/61580644/150752845-005c43e4-793f-464e-a09f-d6491f16f368.png)
![Screen Shot 2022-01-19 at 3 18 40 PM](https://user-images.githubusercontent.com/61580644/150757019-b69908b9-a1ec-4692-a7b5-c3102fe4a8e9.png)
![Screen Shot 2022-01-19 at 3 18 51 PM](https://user-images.githubusercontent.com/61580644/150752851-05223ce2-3e2a-4939-8f73-bd1545c431d7.png)
![Screen Shot 2022-01-19 at 3 19 08 PM](https://user-images.githubusercontent.com/61580644/150752847-7d1a8c96-8866-4b09-9c15-16aa72547b13.png)

**Comment by mikehardy** [comment, marked as answer]
Hey @jpinto321 - does this help? https://github.com/invertase/react-native-firebase/discussions/5848

**Reply by jpinto321** [reply]
@mikehardy yes it does. Thanks
